### PR TITLE
Add ADS-B-Feeder VM

### PR DESCRIPTION
**Pull request by MickLesk**
## Description

New VM: ADS-B-Feeder (Flight Tracker)
Feature Request: https://github.com/tteck/Proxmox/discussions/50#discussioncomment-9316693

VM: 1 Core, 1024 MB RAM, 16GB ROM.
VM Based on DietPi v9.4.1
Only VM possible, because the Files work only with docker-compose. 
VM runs nice (by my tests) 

## Type of change

Please delete options that are not relevant.
- [x] New Script (Develop a new script or set of scripts that are fully functional and thoroughly tested)
- [x] I have performed a self-review of my code, adhering to established codebase patterns and conventions.
- [x] This change requires a documentation update

image
image
image

### CODE REVIEW

File: vm/adsb-feeder-vm.sh [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+  if [ $exitstatus = 0 ]; then
+    if [ -z $MTU1 ]; then
+      MTU1="Default" MTU=""
+      echo -e "${DGN}Using Interface MTU Size: ${BGN}$MTU1${CL}"
+    else
+      MTU=",mtu=$MTU1"
+      echo -e "${DGN}Using Interface MTU Size: ${BGN}$MTU1${CL}"
+    fi
+  fi
+  if (whiptail --backtitle "Proxmox VE Helper Scripts" --title "START VIRTUAL MACHINE" --yesno "Start Mikrotik RouterOS CHR VM when completed?" 10 58); then
```
Comment (tteck) [inline vm/adsb-feeder-vm.sh]: Start Mikrotik RouterOS CHR VM when completed?

**Comment by tteck** [comment]
Is this really worth the potential support headaches?

> note that when running in VMs, there are known issues that are outside of the control of the image with USB timing that frequently lead to MLAT issues.

File: vm/adsb-feeder-vm.sh (hunk 1 above)
Comment (MickLesk) [reply vm/adsb-feeder-vm.sh]: oh lol :D done

**Comment by MickLesk** [comment]
> Is this really worth the potential support headaches?
> 
> > note that when running in VMs, there are known issues that are outside of the control of the image with USB timing that frequently lead to MLAT issues.

I don't know. @mastertape can probably answer that best if he actively uses it. We can also put this on hold and @mastertape can test it with the following:
```
bash -c "$(wget -qLO - https://github.com/MickLesk/Proxmox_DEV/raw/main/vm/adsb-feeder-vm.sh)"
```

**Comment by tteck** [comment]
When a developer states:

> VM setup under VirtualBox (easy), VMware (almost as easy), or Proxmox (**advanced users**) -- note that when running in VMs, there are **known issues** that are outside of the control of the image with USB timing that **frequently lead to MLAT issues.**

We'll put this on _hold_ for now.

**Comment by MickLesk** [comment]
its okay for me 👍🏻

**Comment by mastertape** [comment]
sry was on business trip I will test it during the weekend and give feed back thx!

**Comment by mastertape** [comment]
tried to install the script may be it is to late? it dosent run no installation progress
... sorry that i responce so late - had much trouble last days in real life there was no chance to test it earlier

**Pull request closed without merging**